### ISSUE TITLE
Version 2.19 API changes?

### ISSUE THREAD

**Issue by tjchambers** [elixir-cldr/cldr_dates_times#51]
The public API seems to have changed between version 2.18.1 and 2.19.1. Not suggesting this should mean a major version bump, but IMO it should be noted in the changelog as a change. 

Example: 

```    warning: Cldr.DateTime.Format.date_formats/3 is undefined or private. Did you mean:

          * gmt_format/0
          * time_formats/0
          * time_formats/1
          * time_formats/2
          * time_formats/3

    │
 82 │     {:ok, date_formats} = Cldr.DateTime.Format.date_formats(locale, :gregorian, Sct.Cldr)
    │                                                ~

    │
    └─ lib/sct_web/formatter.ex:82:48: SctWeb.Formatter.default_full_format/1
 ```
    
 I believe this was changed from `DateTime.Format.date_formats` to `Date.formats` . 

There may be similar other API changes.

**Comment by tjchambers** [comment]
As a side comment I believe the docs WERE changed.

**Comment by kipcole9** [comment]
@tjchambers sorry for the inconvenience, there is no intent to change the public API so definitely a bug. I've fixed that and published [ex_cldr_datetimes 2.19.2](https://hex.pm/packages/ex_cldr_date_times/2.19.2) with the following changelog entry:

### Bug Fixes

* Reinstate `Cldr.DateTime.Format.date_formats/3` which was incorrectly removed in version 2.19. Version 2.19 made efforts to improve the symmetry of `Cldr.Date`, `Cldr.Time` and `Cldr.DateTime`. Part of that work is to have each of those modules contain the functions `formats/3` and `available_formats/3`. It was not intended at this time that the equivalent functions be removed from `Cldr.DateTime.Format`. Thanks to @tjchambers for the report. Closes #51.

Thanks again for the report.
